.github/workflows/tests-stable.yaml

```diff
@@ -36,7 +36,7 @@ jobs:
             --privileged \
             --name "buildkitd_${GITHUB_RUN_ID}_${SUFFIX}" \
             -v /run/buildkit:/run/buildkit:rw \
-            moby/buildkit:v0.18.1;
+            moby/buildkit:v0.18.2;
```

Comment:
Shouldn't this now be switched to `:latest`?

Reply:
As discussed offline, these will automatically be bumped by dependabot and this ensures safety between go.mod updates and dependabot updates.